src/languages/en.js

```diff
@@ -146,6 +147,13 @@ export default {
         split: ({amount}) => `Split ${amount}`,
         choosePaymentMethod: 'Choose payment method:',
     },
+    reportDetailsPage: {
+        notificationPreferencesDescription: 'How often should we notify you when there are new messages to catch up on in this room?',
+        always: 'Always',
+        daily: 'Daily',
+        mute: 'Mute',
+        members: 'Members',
+    },
```

Comment:
Spanish translations for these were not included in the Design Doc. Reaching out in #espanol for help.